css/footer.css

```diff
@@ -16,3 +16,124 @@
   cursor: pointer;
   background-color: #008552;
 }
+
+form {
+  padding: 24px;
+}
+
```

Comment:
- [ ] Good job team 😃, but you need to fix one point. Kindly make sure in order to lay out the elements on the page you should use `Flexbox` as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox.) 👍